pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fair-mango"
+dynamic = ["version"]
+dependencies = [
+  "numpy>=1.19.5",
+  "pandas>=1.13.1",
+  "scikit-learn>=1.1.3",
+]
+requires-python = ">=3.10"
+maintainers = [
+  {name = "Marc Bresson", email = "marc.bresson@datategy.net"},
+  {name = "Nacer Kraoudir", email = "nacer.kraoudir@datategy.net"},
+]
```

Comment:
Typo in my name and email address :(
name: `Nacer Kroudir`
email: `nacer.kroudir@datategy.net`

Reply:
you have no proof 👀